### PR TITLE
Sort Organizations by Name

### LINKED ISSUE #2190: Order organizations list by name rather than by ID
https://dmoj.ca/organizations/

### PR DESCRIPTION
**Pull request by jazyz**
Closes #2190

### CODE REVIEW

**Review by quantum5 — CHANGES_REQUESTED** [review]
Why are you sorting in Python? This is obviously a database level operation. Imagine what happens when we are forced to paginate the organization page.

File: judge/views/organization.py
Comment (kiritofeng) [inline judge/views/organization.py:102]: Please fix your code style to be consistent with the rest of our codebase. You can check this locally (without having to wait for us to approve the linter run) by running `flake8` yourself.

**Comment by jazyz** [comment]
Now sorted in database level

**Review by Riolku — APPROVED** [review]
LGTM assuming CI passes

**Review by Ninjaclasher — APPROVED** [review]
LGTM

**Review by quantum5 — APPROVED** [review]

**Pull request merged**
